src/input/touch.c

```diff
@@ -24,6 +24,16 @@ static struct wlr_surface*
 touch_get_coords(struct seat *seat, struct wlr_touch *touch, double x, double y,
 		double *x_offset, double *y_offset)
 {
+	/* Do not return a surface when mouse emulation is enforced. Not
```

Comment:
nit: First comment line in a c style multi line comment should be empty, see e.g. https://github.com/labwc/labwc/blob/master/src/server.c#L233-L239